src/siolib.c

```diff
@@ -0,0 +1,27 @@
+#include "includes/siolib.h"
+#include "includes/sapi.h"
+#include "includes/sparser.h"
+
+#include <stdio.h>
+
+static Constant sglExit(VM* vm, int arity, Constant* args) {
+  int exitCode = NUM_CONST(args[0]);
+
+  exit(exitCode);
+
+  return NULL_CONST;
+}
+
+static Constant sglSystem(VM* vm, int arity, Constant* args) {
+  char* command = AS_CSTRING(args[0]);
+
+  return NUM_CONST(system(command));
+}
```

Comment:
I've got another plan for running commands, for now can you please remove the system command?